content/docs/guide/getting-started.md

```diff
@@ -21,8 +25,37 @@ file "/tmp/mgmt-hello-world" {
 
 Run: `mgmt run --tmp-prefix lang hello0.mcl` to try out this simple example!
 
+Let's break down the file you just created:
+    - `file "/tmp/mgmt-hello-world"` - declares a file resource at the specified path
+    - `content =>` - sets the file's contents
+    - `state => $const.res.file.state.exists` - ensures the file exists (this will be further explained at.... link to guide/link to docs)
+
+
+> **Note: what is .mcl?**  
+> `.mcl` files contain mgmt's configuration language code. MCL (mgmt Configuration Language) is mgmt's domain-specific language for declaring desired system states.
+
+> **Note: Where does `--tmp-prefix lang` come from?**
+> `--tmp-prefix` creates temporary directories for mgmt's runtime files, and `lang` specifies we're using the MCL language parser.
+
+
 # Playing with `mgmt`
 
 While `mgmt` is running, you can test out one of its cool side effects: Run
 `rm /tmp/mgmt-hello-world` in a second terminal, and watch the file come back!
 You can type `^C` to shutdown `mgmt`.
+
+While mgmt is running, you can test out one of its cool side effects: Run rm /tmp/mgmt-hello-world in a second terminal, and watch the file come back! (Note how fast this is done) You can type ^C to shutdown mgmt.
+
+> **Why is `mgmt` different?**
+> This demo showcases mgmt's **real-time** configuration management. Unlike traditional tools that run periodically, mgmt continuously monitors and maintains your desired state instantly when changes occur.
+
+## What's Next?
```

Comment:
Good idea to put this here, but I'll ask the web dev to make a TOC kind of thing so we don't have to duplicate it and keep it in sync.